### DISCUSSION TITLE
How do I create a registration page?

### DISCUSSION THREAD

**Discussion by thanhtoan1196** [filamentphp/filament#2085] [Help] [answered]
How do I create a registration page like this?
https://filamentphp.com/admin/register

**Comment by ryangjchandler** [comment, marked as answer]
@thanhtoan1196 Breezy is being used for that, https://filamentphp.com/plugins/breezy

**Reply by thanhtoan1196** [reply]
thank you

**Reply by JohnRoux** [reply]
Just on this, I don't think breezy has it's own registration page anymore?

Is there an easy way to add registration to a filament panel?

**Reply by leandrocfe** [reply]
Filament V3 you can add in the PanelProvider
`$panel->registration()`

**Reply by JohnRoux** [reply]
Ahhh neat! I didn't see that anywhere in the docs.
Are there any doc resources for that? Getting email verification issues with it

**Reply by JohnRoux** [reply]
Ahhh found it.
Looks like I was missing the email verification one as well :see_no_evil: 

Interesting that searching for registration didn't pull this up at all in the doc search!
https://filamentphp.com/docs/3.x/panels/users#authentication-features
